pkg/cli/app.go

```diff
@@ -0,0 +1,89 @@
+package cli
```

Comment:
Why does some stuff go into `pkg/cli` and other stuff in `cli`? How are you thinking about the `pkg` folder?

Reply:
> Are we using it to contain wrappers of other (e.g., open source) packages? If so, is `pkg` a Go idiom for that or something we came up with here?

Right. I should have explained it.  Added `/pkg/pkg.go` file with comment:

```go
// The purpose of this package `/pkg` is a developing a new common libraries which can be moved to `go-commons` in the future.
// This approach helps us test new functionality well on a single application first and after that to safely merged with rest common libraries in `go-commons`
```